lib/dc/store/aws_s3_store.rb

```diff
@@ -14,13 +14,14 @@ module AwsS3Store
 
       # 60 seconds for persistent connections.
       S3_PARAMS       = {:connection_lifetime => 60}
+      AWS_REGION      = DC::CONFIG['aws_region']
 
       ACCESS_TO_ACL   = Hash.new(:private)
       DC::Access::PUBLIC_LEVELS.each{ |level| ACCESS_TO_ACL[level] = :public_read }
 
       module ClassMethods
         def asset_root
-          "https://s3.amazonaws.com/#{BUCKET_NAME}"
+          "https://s3-#{AWS_REGION}.amazonaws.com/#{BUCKET_NAME}"
```

Comment:
@DavidLemayian I merged this in last week but only this morning actually deployed it, and subsequently blew up image references because this subdomain triggered `ERR_CONNECTION_REFUSED`. (Compare https://s3.amazonaws.com/s3.documentcloud.org/documents/282753/pages/lefler-thesis-p1-normal.gif and https://s3-us-east-1.amazonaws.com/s3.documentcloud.org/documents/282753/pages/lefler-thesis-p1-normal.gif.)

I reverted this line in 56a6a28f440ee78f7c24ac600a36fd3291c86f7c until we can figure out why. I'm presuming it's something easy and probably configurable within our AWS account but, again, I'm pretty AWS ignorant. Any ideas?